src/main/java/com/example/hibernate/controller/CarController.java

```diff
@@ -45,7 +45,7 @@ public ResponseEntity<Car> getCarById(@PathVariable("id") UUID id) {
         }
     }
 
-    @PostMapping("cars")
+    @PostMapping("/cars")
```

Comment:
On your controller you should add mapping `/api/car`, and then you'll put here `@PosstMapping`, just like we did with `addPerson` in demo project